Dockerfile-maya2016

```diff
@@ -5,22 +5,20 @@ RUN wget https://bootstrap.pypa.io/get-pip.py && \
 	mayapy -m pip install \
 		nose \
 		nose-exclude \
-		coverage \
 		pyblish-base==1.4.2 \
 		pyblish-maya \
 		pymongo \
-		# Sphinx 1.8.0 fail to build doc, pin version to 1.7.9
-		# see https://github.com/sphinx-doc/sphinx/issues/5417
-		sphinx==1.7.9 \
+		sphinx \
 		six \
 		sphinxcontrib-napoleon \
-		python-coveralls
+		coveralls
 
 # Avoid creation of auxilliary files
 ENV PYTHONDONTWRITEBYTECODE=1
+ENV TRAVIS=1
 
 WORKDIR /workspace
 
 ENTRYPOINT \
 	PYTHONPATH=$(pwd):$PYTHONPATH mayapy -u run_maya_tests.py && \
-	mayapy -c "import sys, os, coveralls;coveralls.wear() if os.getenv('TRAVIS_JOB_ID') else sys.stdout.write('Skipping coveralls')"
```

Comment:
You're removing some critical information here. Travis needs that ID from the environment, which I've put into the Travis backend environment for passing into it like this.

Could we not keep the command as-is?

Reply:
> Could we not keep the command as-is?

Sadly we couldn't. :(
The interface of [`python-coveralls`](https://github.com/z4r/python-coveralls) was different from [`coveralls`](https://github.com/coveralls-clients/coveralls-python) package, `coveralls` does not have `wear` function exposed.

And since I have added `TRAVIS=1` into environment, so the job id should be able to pick up from [here](https://github.com/coveralls-clients/coveralls-python/blob/c338cab478f67cded2e22d0052064e3135e48bc9/coveralls/api.py#L123) and then [here](https://github.com/coveralls-clients/coveralls-python/blob/c338cab478f67cded2e22d0052064e3135e48bc9/coveralls/api.py#L101). Which it did work on my fork build, see [here](https://travis-ci.org/davidlatwe/core/builds/626053466#L582) :
```
Submitting coverage to coveralls.io...
Coverage submitted!
Job #158.1
https://coveralls.io/jobs/56927382
```

I have also tested with another branch that only pin `python-coveralls` version, and it also able to send report, see [here](https://travis-ci.org/davidlatwe/core/builds/627046213#L542) :
```
INFO:coveralls:200
INFO:coveralls:{"message":"Job #160.1","url":"https://coveralls.io/jobs/57015951"}
```

Tricky CI :face_with_head_bandage: